discrete_summary_checker/example/Check_Format.ipynb

```diff
@@ -0,0 +1,956 @@
+    "Next, need to check that all of the metadata columns are the same for each station. We do this on a station-by-station basis."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "metadata_columns = [\"Cruise\", \"Station\", \"Target Asset\", \"Start Latitude [degrees]\", \n",
+    "                    \"Start Longitude [degrees]\", \"Start Time [UTC]\", \"Cast\",\n",
+    "                   \"Bottom Depth at Start Position [m]\", \"CTD File\"]\n",
+    "metadata = summary_sheet[metadata_columns]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Metadata Schema\n",
+    "Build the schema for validating the metadata information is all the same for each station."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Rebuild the schema, grouping by input\n",
+    "metadata_schema = Schema([\n",
+    "    Column(\"Cruise\", [IsSameValidation]),\n",
+    "    Column(\"Station\", [IsSameValidation]),\n",
+    "    Column(\"Target Asset\", [IsSameValidation]),\n",
+    "    Column(\"Start Latitude [degrees]\", [IsSameValidation]),\n",
+    "    Column(\"Start Longitude [degrees]\", [IsSameValidation]),\n",
+    "    Column(\"Start Time [UTC]\", [IsSameValidation]),\n",
+    "    Column(\"Cast\", [IsSameValidation]),\n",
+    "    Column(\"Bottom Depth at Start Position [m]\", [IsSameValidation]),\n",
+    "    Column(\"CTD File\", [IsSameValidation]),\n",
+    "])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run the validation on a station-by-station basis:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for station in metadata[\"Station\"].unique():\n",
+    "    # Get the data associated with a particular station\n",
+    "    station_data = metadata[metadata[\"Station\"] == station]\n",
+    "    \n",
+    "    # Run it through the validation checker\n",
+    "    merrors = metadata_schema.validate(station_data)\n",
+    "    for error in merrors:\n",
+    "        print(error)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "jupytext": {
+   "formats": "ipynb,py:light"
+  },
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.8"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
Ah... this is a Jupyter notebook that runs the Check_Format.py code!  Got it.  Looks good.